index.js

```diff
@@ -60,13 +60,54 @@ animalToRemove.remove()
 
 /***** Deliverable 1 *****/
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
-}
+console.log("BigRich")}
+
+header.addEventListener('click', function(){
+
+    toggleColor(header)
+
+})
```

Comment:
Great job using the `toggleColor` function! You can also pass the event target as an argument to the function as well.

We have the header variable already defined earlier in the file.
All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.


```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})

```

With ES6 notation

```
header.addEventListener("click", e => toggleColor(e.target))
```